docs/development/ide-configuration.md

```diff
@@ -107,12 +107,8 @@ It is as simple as doing `right-click` on the file `.eslintrc.js` and choosing o
 
 #### [Jest tests](https://www.jetbrains.com/help/idea/running-unit-tests-on-jest.html)
 
-To be able to run tests from IntelliJ, you must set up the default Jest template in `Run/Debug Configurations`
-
-Adjust following parameters:
-- Configuration files: it depends on the type of tests you want to run 
-  - unit tests: `<project_dir>/jest.config.unit.js`
-  - end to end tests: `<project_dir>/jest.config.e2e.js`
+You can run tests directly from IntelliJ - there is no need for any additional configuration.
```

Comment:
```suggestion
You can run tests directly from IntelliJ - there is no need for any additional configuration. \
```

Reply:
Nicely spotted,
I have added new line (by putting double space at the end of line)